recipes/openssh/all/conanfile.py

```diff
@@ -66,6 +66,10 @@ def validate(self):
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        if self.version in ["9.1p1", "9.6p1"]:
```

Comment:
version 9.1p1 would have not accepted version 3.2,
and version 9.6p1 would have not accepted versions higher than than 3.2, so the recipe would've started failing once OpenSSL 3.3 or higher was released. Newer versions should already have the fix to accept any in the 3.x range, as per the OpenSSL versioning guidelines